cmd/osm-webhook/osm-webhook.go

```diff
@@ -0,0 +1,263 @@
+// Command osm-webhook starts up a Kubernetes Validating Webhook on the
```

Comment:
I see a lot of logic here is duplicated from what exists in the osm-controller, is there a specific reason why a separate pod was created for the webhook and why can't this be a part of osm-controller ?